src/Test.UnitTests.Sarif/Readers/DeferredCollectionsTests.cs

```diff
@@ -126,6 +156,34 @@ private static void CompareReadNormalToReadDeferred(string filePath)
             Assert.Equal(contextsCopy.Count, valueCount);
         }
 
+        private static void CompareReadNormalToReadDeferred(string filePath)
+        {
+            LogModelSampleBuilder.EnsureSamplesBuilt();
+            var serializer = new JsonSerializer();
+
+            Log expected;
+            Log actual;
+
```

Comment:
In future, waterfall from shortest line to longest, that's our convention.

Log actual;
Log expected;